assets/scripts/build_gallery.py

```diff
@@ -117,12 +144,33 @@ def get_expectation_file_info_dict(
 
     for file_path in sorted(files_found):
         file_path = file_path.replace(f"{repo_path}{os.path.sep}", "")
-        package_name = os.path.basename(os.path.dirname(os.path.dirname(file_path)))
-        if package_name == "expectations":
-            package_name = "core"
         name = os.path.basename(file_path).replace(".py", "")
```

Comment:
Just a small suggestion for readability.
```suggestion
        expectation_name = os.path.basename(file_path).replace(".py", "")
```

Reply:
I'll change it to "expectation" to be consistent with how it's called in other places.

Reply:
I think that is confusing. You also have the expectation implementation as well. This variable is referring to a string that represents the name of the expectation. So this should be called `expectation_name` and the expectation implementation should be called `expectation`

Reply:
The Expectation implementation is not saved anywhere. An instance is immediately created and added to the dict via `great_expectations.expectations.registry.get_expectation_impl(expectation)()`

I renamed expectation to expectation_name and instance to expectation_instance.